package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "sl-sunbird-services",
+  "version": "1.0.0",
+  "description": "sunbird services",
+  "main": "app.js",
+  "bin": {
+    "sunbird-migrations": "module/migrations/bin/migrations.js"
+  },
+  "scripts": {
+    "test": "mocha --timeout 10000",
+    "start": "node app.js",
+    "dev": "nodemon app.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/shikshalokam/sl-sunbird-services.git"
+  },
+  "keywords": [
+    "shikshalokam"
+  ],
+  "author": "Shikshalokam <info@shikshalokam.com>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/shikshalokam/sl-sunbird-services/issues"
+  },
+  "homepage": "https://github.com/shikshalokam/sl-sunbird-services#readme",
+  "dependencies": {
+    "@azure/storage-blob": "^12.2.0-preview.1",
+    "@elastic/elasticsearch": "^7.8.0",
+    "@google-cloud/storage": "^5.1.2",
+    "adm-zip": "^0.4.14",
+    "async": "^3.2.0",
+    "aws-sdk": "^2.642.0",
+    "body-parser": "^1.19.0",
+    "bunyan": "^1.8.12",
+    "bunyan-format": "^0.2.1",
+    "cache-manager": "^3.1.0",
+    "cli-table": "^0.3.1",
+    "cors": "^2.8.5",
+    "csvtojson": "^2.0.10",
+    "dotenv": "^8.2.0",
+    "ejs": "^3.1.3",
+    "express": "^4.17.1",
+    "express-cassandra": "^2.3.2",
+    "express-fileupload": "^1.1.6",
+    "express-validator": "^5.3.1",
+    "fn-args": "^4.0.0",
+    "form-urlencoded": "^4.1.3",
+    "fs": "0.0.1-security",
```

Comment:
@rakeshSgr - I think one fs-extra module is used in migrations, which is not included here